persistence/driver/postgres/schema.go

```diff
@@ -0,0 +1,69 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+)
+
+// CreateJournalSchema creates a PostgreSQL schema for storing journal records.
+func CreateJournalSchema(
+	ctx context.Context,
+	db *sql.DB,
+) error {
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback() // nolint:errcheck
+
+	if _, err := db.ExecContext(ctx, `CREATE SCHEMA IF NOT EXISTS veracity`); err != nil {
+		return err
+	}
+
+	if _, err := db.ExecContext(
+		ctx,
+		`CREATE TABLE IF NOT EXISTS veracity.journal (
+			key     TEXT NOT NULL,
+			version BIGINT NOT NULL,
+			record  BYTEA NOT NULL,
+
+			PRIMARY KEY (key, version)
+		)`,
+	); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// CreateKVStoreSchema creates a PostgreSQL schema for storing key-value store
+// records.
+func CreateKVStoreSchema(
+	ctx context.Context,
+	db *sql.DB,
+) error {
+	tx, err := db.Begin()
+	if err != nil {
+		return err
+	}
+	defer tx.Rollback() // nolint:errcheck
+
+	if _, err := db.ExecContext(ctx, `CREATE SCHEMA IF NOT EXISTS veracity`); err != nil {
+		return err
+	}
+
+	if _, err := db.ExecContext(
+		ctx,
+		`CREATE TABLE IF NOT EXISTS veracity.kvstore (
+			keyspace TEXT NOT NULL,
```

Comment:
```suggestion
		`CREATE TABLE IF NOT EXISTS veracity.keyspace (
			path TEXT NOT NULL,
```

I've been struggling with this naming here, but I think it'd be good if the schema more closely matched the code, in general. I'll take a look at the `DynamoDB` implementation as well.

Reply:
Here are my suggestions for making the journal implementations use the same naming style https://github.com/dogmatiq/veracity/pull/274